Makefile

```diff
@@ -20,3 +20,6 @@ gofmt:
 	./hack/gofmt-all.sh
 
 verify: verify-gofmt test
+
+run:
```

Comment:
call this `run-sample` or something, if we have this.
In general, though, while it's possible to run the adapter outside of a pod, it's not going to run out-of-the-box without some flags.  You're generally better of running it in-cluster.

Reply:
Could we provide an example of how to run it in both scenarios?  I would think running it outside the cluster would make it easier to debug during the development cycle.

What are the flags to run it out-of-the-box?

Reply:
you'd need to pass in kubeconfigs in the right places (you need the main one, plus additional ones for auth, IIRC).

Reply:
I tried to do this but didn't i guess I am still missing the correct configs.  Here is what I passed:

` ./_output/amd64/sample-adapter --secure-port=6443  --authentication-kubeconfig="/home/jstur/.kube/config"  --lister-kubeconfig="/home/jstur/.kube/config"`

What am I missing?